### ISSUE TITLE
request: command option count for jfrog rt s

### ISSUE THREAD

**Issue by sdrzazga** [jfrog/jfrog-cli#319]
Hi.
It would be great if there was a command option for Artifactory's search which would return for the given search criteria the only the number of found artifacts instead of JSON result.

Example:
```
$ jfrog rt s --count-only "frog-repo/rabbit/*.zip"
$ 42
```
Thanks in advance!

**Comment by eyalbe4** [comment]
@sdrzazga,
You can easily count wuth the help of [jq](https://stedolan.github.io/jq/manual/).
Here's how you do it:
```
$ jfrog rt s "frog-repo/rabbit/*.zip" | jq length
[Info] Searching artifacts...
[Info] Found 42 artifacts.
42
```
Only *42* is printed into stdout, so you don't need to worry about the Info logging.
Please let me know if this helps.

**Comment by sdrzazga** [comment]
@eyalbe4 
It is certainly one way to solve it.
However, it always requires a dependency on another program to do this job and additional piping which reduces readability. Also, piping doesn't work on windows system unless they have gnu tools installed. It is also not installed in the docker image for jfrog-cli.
Anyways, I would find it the easiest and most versatile option if it was implemented in frog-cli itself since you have the information there already instead of relying on something external.
Wouldn't you agree?

**Comment by eyalbe4** [comment]
Agree @sdrzazga - 
@AlexeiVainshtein created https://github.com/jfrog/jfrog-cli/pull/412 to allow this.

**Comment by sdrzazga** [comment]
I believe this was implemented with 1.27.0 (Aug 16, 2019 - https://bintray.com/jfrog/jfrog-cli-go/jfrog-cli-linux-amd64#release):

"New --count option added to the search command."
